src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1649,6 +1645,14 @@ private bool AddOrganelle(string organelleType)
                     {
                         usedHexes.Add(hex);
                     }
+
+                    if (organelleDefinition.CanBeModified &&
+                        Symmetry == HexEditorSymmetry.None &&
+                        (TutorialState == null || !TutorialState.Enabled || TutorialState.EditorUndoTutorial.Complete))
```

Comment:
This feels like a pretty ugly way to check the tutorial state is safe. Is there a better way to do it that I should be aware of? Or should this section be placed somewhere else entirely to avoid it?

Reply:
There is `TutorialState.TutorialActive` to see if any tutorial is currently open.

Reply:
Removed anyway now so no longer relevant,